src/test/llmtemplate_test.cpp

```diff
@@ -348,3 +355,299 @@ TEST_F(LLMChatTemplateTest, ChatTemplateTwoConfigs) {
     ASSERT_EQ(applyChatTemplate(nodeResources->textProcessor, nodeResources->modelsPath, payloadBody, finalPrompt), true);
     ASSERT_EQ(finalPrompt, expectedOutput);
 }
+
+std::string configTemplate = R"(
+        {
+            "model_config_list": [],
+            "mediapipe_config_list": [
+            {
+                "name":"llmDummyKFS",
+                "graph_path":"GRAPH_PATTERN"
+            }
+            ]
+        }
+    )";
+
+std::string graphTemplate = R"(
+        input_stream: "HTTP_REQUEST_PAYLOAD:input"
+        output_stream: "HTTP_RESPONSE_PAYLOAD:output"
+        node {
+            name: "llmNode1"
+            calculator: "HttpLLMCalculator"
+            input_side_packet: "LLM_NODE_RESOURCES:llm"
+            input_stream: "LOOPBACK:loopback"
+            input_stream: "HTTP_REQUEST_PAYLOAD:input"
+            output_stream: "LOOPBACK:loopback"
+            output_stream: "HTTP_RESPONSE_PAYLOAD:output"
+            input_stream_info: {
+            tag_index: 'LOOPBACK:0',
+            back_edge: true
+            }
+            node_options: {
+                [type.googleapis.com/mediapipe.LLMCalculatorOptions]: {
+                models_path: "MODELS_PATTERN",
+                }
+            }
+            input_stream_handler {
+            input_stream_handler: "SyncSetInputStreamHandler",
+            options {
+                [mediapipe.SyncSetInputStreamHandlerOptions.ext] {
+                sync_set {
+                    tag_index: "LOOPBACK:0"
+                }
+                }
+            }
+            }
+    })";
+
+class CleanupFilesGuard {
+
+public:
+    CleanupFilesGuard(){}
+    ~CleanupFilesGuard() {
+        ovmsServer.shutdownModules();
+    }
+};
+
+class LLMChatTemplateHttpTest : public TestWithTempDir {
+protected:
+    static std::unique_ptr<std::thread> t;
+
+    const std::string GRAPH_PATTERN = "GRAPH_PATTERN";
+    const std::string WORKSPACE_PATTERN = "MODELS_PATTERN";
```

Comment:
```suggestion
    const std::string GRAPH_PATTERN = "<GRAPH_PATTERN>";
    const std::string WORKSPACE_PATTERN = "<MODELS_PATTERN>";
```
In other places we use put parts that we want to substitute later in `<>`